stages/org.osbuild.ostree

```diff
@@ -180,15 +185,46 @@ def make_fs_identifier(desc):
     raise ValueError("unknown rootfs type")
 
 
-def main(tree, sources, options):
+def populate_var(sysroot):
+    # Like anaconda[1] and Fedora CoreOS dracut[2]
+    # [1] pyanaconda/payload/rpmostreepayload.py
+    # [2] ignition-ostree-populate-var.sh
```

Comment:
This is some ugly technical debt to have copied into a new place.  I think we've gotten very close to dropping this for FCOS, but not RHCOS yet due to some SELinux issues.